### PR TITLE
7439 - 5.3 release notes

### LINKED ISSUE #7439: 5.3 Release Notes
It's that time again.

### PR DESCRIPTION
**Pull request by djbrooke**
**What this PR does / why we need it**:

The release notes for the 5.3 pre-holiday break release spectacular. 🎁 

**Which issue(s) this PR closes**:

Closes #7439

**Special notes for your reviewer**:

- Please check my usage of "obsolete" in regards to the DB settings for banner management. Not sure if it should be obsolete or deprecated, or something else. 
- ~~Please check where I put the Payara upgrade in the release steps. I put it after "Undeploy the Previous Version"~~ this was handled in the updates for a later release note from @poikilotherm 

**Suggestions on how to test this**:

Lint check, grammar/spelling check. :) 

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

N/A 

**Is there a release notes update needed for this change?**:

Meta

**Additional documentation**:

N/A

### CODE REVIEW

**Comment by djbrooke** [comment]
Converting to a draft PR, comments very much welcome. I think we're potentially waiting on:

- #7398
- #7426/#7406 (likely one PR)
- #7422 (lots of changes here)

File: doc/release-notes/5.3-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- `<payara install path>/bin/asadmin deploy dataverse-5.3.war`
+
+5\. Restart payara
+
+- `service payara stop`
+- `service payara start`
+
+6\. Reset the EJB timer database back to default:
+
+- `<payara install path>/asadmin set configs.config.server-config.ejb-container.ejb-timer-service.timer-datasource=jdbc/__TimerPool`
```
Comment (scolapasta) [inline doc/release-notes/5.3-release-notes.md:96]: I think this needs a /bin/

File: doc/release-notes/5.3-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+(where `<payara install path>` is where Payara 5 is installed, for example: `/usr/local/payara5`)
+
+2\. Upgrade to Payara 5.2020.6 following the steps in the [Payara documentation](https://docs.payara.fish/community/docs/5.2020.6/documentation/user-guides/upgrade-payara.html)
+
+3\. Stop payara and remove the generated directory, start.
+
+- `service payara stop`
+- remove the generated directory:
+`rm -rf <payara install path>/payara/domains/domain1/generated`
```
Comment (scolapasta) [inline doc/release-notes/5.3-release-notes.md:82]: I think this should be glassfish and not payara.

File: doc/release-notes/5.3-release-notes.md [hunk 3]
```diff
@@ -44,7 +54,28 @@ The functionality previously provided by the DB settings :StatusMessageHeader an
 
 ### New Database Settings and JVM Options
 
-No new DB Settings or JVM Options have been added in this release. See above note about obsolete options.
+Several new DB options have been added in this release:
```
Comment (poikilotherm) [inline doc/release-notes/5.3-release-notes.md:57]: "DB options" might be mistaken for "database-based settings" if not put into more context.

File: doc/release-notes/5.3-release-notes.md (hunk 1 above)
Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: added in https://github.com/IQSS/dataverse/commit/232751af9f6be3c2ffcaee01c655aefe53a0d658

File: doc/release-notes/5.3-release-notes.md (hunk 2 above)
Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md:139]: @scolapasta I'll do what you tell me here, but it looks like we've been switching it back and forth.  :) 
 
5.1:
`- remove the generated directory: rm -rf <payara install path>payara/payara/domains/domain1/generated`

5.1.1:
`- remove the generated directory:
`rm -rf <payara install path>/glassfish/domains/domain1/generated``

5.2:
`- remove the generated directory: 
`rm -rf <payara install path>/payara/domains/domain1/generated``

Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: updated in https://github.com/IQSS/dataverse/commit/5717a4024129a54a81e9c38d80fe4e3954388ad2

File: doc/release-notes/5.3-release-notes.md (hunk 3 above)
Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: clarified these are JVM options in https://github.com/IQSS/dataverse/commit/9b875614cf006e0d7b182e4aca8f7ddaf5fd3a57

**Comment by djbrooke** [comment]
@scolapasta @poikilotherm and others - ready for more review here. Thanks for the reviews so far!

**Review by poikilotherm — COMMENTED** [review]
LGTM,

It would be great if we can add #7457 / #7463 to this release, as it explains a lot about the why and how related to MPCONFIG.
@pdurbin was already satisfied and this doesn't change a thing for anyone, as this is groundwork to enable further steps.

File: doc/release-notes/5.3-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+The Ingest process uses the Java Message System to create ingest tasks in a queue. That queue had been configured from command line or domain.xml before. This has now changed to being done
+in code.
+
+In the unlikely case you might want to change any of these settings, feel free to change and recompile or raise an issue on Github. See `IngestQueueProducer` for more details.
+
+If you want to clean up your existing installation, you can delete the old, unused queue like this:
+
+```shell
+asadmin delete-connector-connection-pool --cascade=true jms/IngestQueueConnectionFactoryPool
```
Comment (scolapasta) [inline doc/release-notes/5.3-release-notes.md:80]: for consistency, this should like the asadmin directions below, i.e:
- `<payara install path>/asadmin

File: doc/release-notes/5.3-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+Please configure your connection details, replacing all the `${DB_...}`.
+(If you are using a PostgreSQL server on `localhost:5432`, you can omit `dataverse.db.host` and `dataverse.db.port`.)
+
+- `<payara install path>/asadmin create-system-properties "dataverse.db.user=${DB_USER}"`
+- `<payara install path>/asadmin create-system-properties "dataverse.db.host=${DB_HOST}"`
+- `<payara install path>/asadmin create-system-properties "dataverse.db.port=${DB_PORT}"`
+- `<payara install path>/asadmin create-system-properties "dataverse.db.name=${DB_NAME}"`
+- `echo "AS_ADMIN_ALIASPASSWORD=${DB_PASS}" > /tmp/password.txt`
+
+- `<payara install path>/asadmin create-password-alias --passwordfile /tmp/password.txt dataverse.db.password`
```
Comment (scolapasta) [inline doc/release-notes/5.3-release-notes.md:126]: these are missing /bin/ before asadmin

File: doc/release-notes/5.3-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+- `rm /tmp/password.txt`
+
+4\. Reset the EJB timer database back to default:
+
+- `<payara install path>/bin/asadmin set configs.config.server-config.ejb-container.ejb-timer-service.timer-datasource=jdbc/__TimerPool`
+
+5\. Delete the old password alias and DB pool:
+
+- `<payara install path>/asadmin delete-jdbc-connection-pool --cascade=true dvnDbPool`
+- `<payara install path>/asadmin delete-password-alias db_password_alias`
```
Comment (scolapasta) [inline doc/release-notes/5.3-release-notes.md:136]: these are missing /bin/ before asadmin

**Review by scolapasta — CHANGES_REQUESTED** [review]
some minor changes, adding /bin/ where needed, and a consistency change

File: doc/release-notes/5.3-release-notes.md (hunk 5 above)
Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: added in https://github.com/IQSS/dataverse/commit/f82f970aa6e1706dcf631eebcb498c159f6ffc96

File: doc/release-notes/5.3-release-notes.md (hunk 6 above)
Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: added in https://github.com/IQSS/dataverse/commit/f82f970aa6e1706dcf631eebcb498c159f6ffc96

File: doc/release-notes/5.3-release-notes.md (hunk 4 above)
Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: reformatted in https://github.com/IQSS/dataverse/commit/f82f970aa6e1706dcf631eebcb498c159f6ffc96

@scolapasta - should I add /bin here as well?

Comment (scolapasta) [reply doc/release-notes/5.3-release-notes.md]: yes, whoops meant to add that there...

Comment (djbrooke) [reply doc/release-notes/5.3-release-notes.md]: added in https://github.com/IQSS/dataverse/pull/7446/commits/faf0e2b13f70cd46a82a8a883f1e1cf2533d8d8f

**Review by scolapasta — APPROVED** [review]
Looks ready for release!

**Pull request merged**